qa/rpc-tests/nuparams.py

```diff
@@ -72,12 +74,15 @@ def run_test(self):
 
         nu5 = upgrades[nustr(NU5_BRANCH_ID)]
         assert_equal(nu5['name'], 'NU5')
-        assert_equal(nu5['activationheight'], 5)
+        assert_equal(nu5['activationheight'], 7)
         assert_equal(nu5['status'], 'pending')
 
+        # Initial subsidy at the genesis block is 12.5 ZEC
+        assert_equal(node.getblocksubsidy()["miner"], Decimal("12.5"))
```

Comment:
On regtest, yes. (On testnet and mainnet, the miner subsidy at the genesis block is zero because of slow start.)

I'd forgotten that there is no Founders' Reward output in the genesis block, but [section 7.9 of the spec](https://zips.z.cash/protocol/protocol.pdf#foundersreward) is clear that there isn't.